.github/workflows/sycl-detect-changes.yml

```diff
@@ -54,6 +54,8 @@ jobs:
               - *libclc
               - 'sycl/*'
               - 'sycl/!(test-e2e|doc)/**'
+            libdevice: &libdevice
+              - 'libdevice/**'
```

Comment:
I think we need a comment explaining why this doesn't depend on anything.

Reply:
will do